src/nwb_datajoint/common/common_ephys.py

```diff
@@ -326,7 +326,8 @@ def make(self, key):
             return None
         # get the list of selected LFP Channels from LFPElectrode
         electrode_keys = (LFPSelection.LFPElectrode & key).fetch('KEY')
-        electrode_id_list = list(k['electrode_id'] for k in electrode_keys).sort()
+        electrode_id_list = list(k['electrode_id'] for k in electrode_keys)
+        electrode_id_list = electrode_id_list.sort()
```

Comment:
I think this still sets `electrode_id_list` to `None`. Either the stuff before and including equal sign must be removed, or we should just use numpy: `electrode_id_list = np.sort(electrode_id_list)` (could just do it in line 329).